uelc/main/models.py

```diff
@@ -611,4 +612,27 @@ class CaseAnswerForm(forms.Form):
         return CaseAnswerForm()
 
 
+class SectionSubmission(models.Model):
```

Comment:
thanks for catching this! It is only supposed to be in GateBlock. I moved it over there from main because I was having issues importing it from uelc.main.models. It was not getting registered and was causing an error on import.